sys/parsoid.js

```diff
@@ -354,7 +354,10 @@ PSP.generateAndSave = function(restbase, req, format, currentContentRes) {
                     }
                 })
                 .catch(function(e) {
-                    self.log('error/summary', e);
+                    if (e.status !== 501) {
```

Comment:
We might want to ignore 404s as well, since they are not technically errors from RESTBase's point of view.